src/puppet-web/browser-driver.ts

```diff
@@ -81,6 +81,8 @@ export class BrowserDriver {
     }
    */
 
+    require('chromedriver')
```

Comment:
Could you please add comments above this line to memo the details?

Becasue it looks wired if without any description.

Anyway, good catch and nice patch. Thank you!